spec.bs

```diff
@@ -1014,23 +1014,35 @@ special value "*". The only interfaces that can be exposed inside of the globals
 specification are those that explicitly list the global names provided here.
 
 <pre class="idl">
-[Exposed=InterestGroupBiddingScriptRunnerGlobalScope,
-Global=InterestGroupBiddingScriptRunnerGlobalScope]
-interface InterestGroupBiddingScriptRunnerGlobalScope {
+[Global=InterestGroupScriptRunnerGlobalScope,

```

Comment:
We don't need to include the `Global` extended attribute on the base interface if it's included elsewhere. Removing it from here would be consistent with worker and worklet specs.